core/controllers/admin.py

```diff
@@ -1066,3 +1066,30 @@ def get(self):
             'number_of_pending_deletion_models': (
                 wipeout_service.get_number_of_pending_deletion_requests())
         })
+
+
+class GetModelsRelatedToUserHandler(base.BaseHandler):
```

Comment:
VerifyUserModelsDeletedHandler* (To avoid get/put.. keyword from the handler name.)